### ISSUE TITLE
Animate a wave function

### ISSUE THREAD

**Issue by Garyfallidis** [fury-gl/fury#324]
Animate a wave function on a surface. You could start with a random 2D function that changes in time. But then make something that has more structure.
Use a colormap to show different levels on the surface.

**Comment by SunTzunami** [comment]
Seems intriguing. Can I work on it?

**Comment by skoudoro** [comment]
yes, go ahead @SunTzunami!  Thank you

**Comment by SunTzunami** [comment]
I've made a function which can animate any 2D function, I figured that there are two ways to go about it- 

1. Plotting the function using lines-  

![2dfuncvialines](https://user-images.githubusercontent.com/65067354/98525195-27a16d80-229e-11eb-92f0-221d6b9fdcfb.gif)

2. Plotting the function using points-

![2dfuncviapoints](https://user-images.githubusercontent.com/65067354/98526251-8b786600-229f-11eb-91a5-ca6059d76f9c.gif)

I can't seem to figure out how to use the colormap properly (haven't been able to display the variations in elevation via the colors)
@skoudoro  @Garyfallidis  I'd be extremely grateful for any feedback/points regarding how to improve the visuals.
